### PR TITLE
Migration of page Home with css

### PR DESCRIPTION
**Pull request by chouaibj**

### CODE REVIEW

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -11,6 +11,8 @@ class HomeController extends AbstractController
     #[Route('/', name: 'app_home')]
     public function index(): Response
     {
-        return $this->render('home/index.html.twig');
+        return $this->render('home/index.html.twig', [
+            'pageName' => 'Home',
```
Comment (Benja-san) [inline src/Controller/HomeController.php]: French for better fit with french content

File: templates/home/index.html.twig [hunk 2]
```diff
@@ -1,15 +1,47 @@
...
-        </h1>
-    </div>
-    <div class="white-stripe">
-        <img src="{{ asset('build/images/logo-wild-code-school.svg') }}"
-             alt="">
+{% block title %} {{ parent() }} {{ pageName }} {% endblock %} {% block stylesheet %}
+<link rel="stylesheet" href="assets/css/banner.css" />
+{% endblock %}
+{% block header %}{% endblock %}
+{% block content %}
```
Comment (Benja-san) [inline templates/home/index.html.twig]: no H1 no main tag

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: templates/home/index.html.twig [hunk 3]
```diff
@@ -1,15 +1,46 @@
...
-        </h1>
-    </div>
-    <div class="white-stripe">
-        <img src="{{ asset('build/images/logo-wild-code-school.svg') }}"
-             alt="">
+{% block title %} {{ parent() }} {{ pageName }} {% endblock %} {% block stylesheet %}
+<link rel="stylesheet" href="assets/css/banner.css" />
+{% endblock %}
+{% block content %}
+<div id="carouselExampleAutoplaying"
```
Comment (Benja-san) [inline templates/home/index.html.twig]: main tag instead of div tag

File: templates/home/index.html.twig [hunk 4]
```diff
@@ -1,15 +1,46 @@
...
+    class="carousel slide"
+    data-bs-ride="carousel">
+    <div class="main-banner">
```
Comment (Benja-san) [inline templates/home/index.html.twig]: use section / ul and li tags for better referencing

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: templates/home/index.html.twig [hunk 5]
```diff
@@ -1,46 +1,59 @@
...
-            </div>
-            <div class="carousel-item">
-                <div class="item item-2">
-                    <div class="header-text">
-                        <h1>A venir...<br /></h1></a>
+                </li>
+                <li class="carousel-item">
+                    <div class="item item-2">
+                        <div class="header-text">
+                            <h1>A venir...<br /></h1>
```
Comment (Benja-san) [inline templates/home/index.html.twig:27]: h2

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by joelmayemba — APPROVED** [review]

**Pull request merged**